beacon-chain/state/state-native/v2/state_trie.go

```diff
@@ -209,7 +246,7 @@ func (b *BeaconState) Copy() state.BeaconState {
 			delete(b.sharedFieldReferences, field)
 			delete(b.stateFieldLeaves, field)
 		}
-		state.StateCount.Sub(1)
+		state.StateCount.Inc()
```

Comment:
What does this do and why are increasing now instead of sub?

Reply:
It's a metric tracking the number of active state objects. This change is a copy-paste mistake. Thanks for catching it!